packages/grafana-data/src/panel/PanelPlugin.ts

```diff
@@ -137,6 +139,10 @@ export class PanelPlugin<TOptions = any, TFieldConfigOptions extends object = an
     configDefaults.custom = {} as TFieldConfigOptions;
 
     for (const option of this.fieldConfigRegistry.list()) {
+      if (option.defaultValue === null || option.defaultValue === undefined) {
+        continue;
+      }
+
       set(configDefaults, option.id, option.defaultValue);
     }
```

Comment:
We where setting props with a default value of undefined, maybe we should set props with a default value of null as there is a distinction there

Reply:
changed this to only ignore props with undefined as default